unit_test/crypto_props_unit_tests.c

```diff
@@ -0,0 +1,53 @@
+/*
+ This file implements unit tests of defined in functions from crypto_props.h
+ */
+/* 
+ * File:   crypto_props_unit_tests.c
+ * Author: Ary Naim (aryan.e.naim@jpl.nasa.gov)
+ *
+ * Created on November 4, 2021, 9:37 AM
+ */
+#include <stdio.h>
+#include <stdlib.h>
+#include "crypto_props.h"
+
+void test()
+{
+    //memory allocation max_key_size=256,max_value_size=4096
+    crypto_config_list* props = crypto_config_alloc(256,4096); 
+    if (NULL!=props)
+    {
+        printf("alloc\n");
+    }
+    //load sample key,values from the file system
+    crypto_config_load_properties("<your_path>/docs/config_props.txt");
```

Comment:
The config_props.txt file will not exist in the test suite.

Reply:
Just FYI because it's not intuitive.  This is something we may move and and refactor later. 
The unit tests currently live in _crypto_util/app/ut_tc_apply.c_.  _ut_tc_apply.c_ is where our unit tests for ApplySecurity() live.  The header-only unit test framework is found at crypto_util/include/utest.h.
We want to simplify this directory structure in the future.

Reply:
@IbraheemYSaleh, Im currently developing this locally and just dropping the my .h,.C, and data files in the CryptoLib headers, src after successful compilation & testing. Not sure how to integrate it in the CryptoLib & run it from there. Lets setup a meeting and run this via CryptoLibs test script. Anyways I will update that path to be relative.